boringtun/src/noise/handshake.rs

```diff
@@ -250,7 +252,7 @@ impl std::fmt::Debug for NoiseParams {
             .field("peer_static_public", &self.peer_static_public)
             .field("static_shared", &"<redacted>")
             .field("sending_mac1_key", &self.sending_mac1_key)
-            .field("preshared_key", &self.preshared_key)
+            .field("preshared_key", &"<redacted>")
```

Comment:
Also fixes a situation here where the preshared key could have accidentially been leaked via the `Debug` impl.